api/alert_channels_jira_cloud_server.go

```diff
@@ -45,7 +73,8 @@ type JiraDataV2 struct {
 	JiraUrl            string `json:"jiraUrl"`
 	ProjectID          string `json:"projectId"`
 	Username           string `json:"username"`
-	Password           string `json:"password,omitempty"` // used for Jira Server
+	Password           string `json:"password,omitempty"`            // used for Jira Server
+	Configuration      string `json:"bidirectionalConfig,omitempty"` // used for bidirectional integration
```

Comment:
Do we need to include the field `bidirectionalWebhook` as well?
```suggestion
	Password             string `json:"password,omitempty"`            // used for Jira Server
	Configuration        string `json:"bidirectionalConfig,omitempty"` // used for bidirectional integration
	BidirectionalWebhook string `json:"bidirectionalConfig,omitempty"`
```

Reply:
`bidirectionalWebhook` only seemed relevant for list as this is generated by Lacework, not something supplied by the user.  From my understanding for list/show we are serializing to a generic interface here:

https://github.com/lacework/go-sdk/blob/bdb63178d5b421bea431c12839974b358d254fc2/api/alert_channels.go#L189